packages/fxa-react/components/LoadingSpinner/en.ftl

```diff
@@ -0,0 +1,4 @@
+## FxA React - Strings shared between multiple FxA products for loading spinner
+
+# Aria label for spinner image indicating data is loading
+app-loading-spinner-aria-label-loading = Loading…
```

Comment:
This could be refactored to this format, used in other packages:

```
# Aria label for loading spinner
app-loading-spinner-aria-label-loading =
  .aria-label = Loading…
```

in component, could look like this (simplified)
```
<Localized id="app-loading-spinner-aria-label-loading">
  <Spinner
    className="w-10 h-10 animate-spin"
    role="img"
    aria-label="Loading…"
  />
</Localized>
```

Reply:
Will keep current format to avoid retranslation